.changeset/lovely-foxes-bow.md

```diff
@@ -0,0 +1,14 @@
+---
+'@shopify/cli-kit': minor
+'@shopify/create-hydrogen': minor
+'@shopify/cli-hydrogen': patch
+---
+
+- cli-hydrogen:
```

Comment:
We are not mentioning the `shopify hydrogen deploy` command in the release notes since its hidden & restricted to Shopify staff only at the moment. Once we remove those conditions we will add it as part of the release notes.